src/utilities/astFromValue.js

```diff
@@ -52,6 +54,7 @@ import { GraphQLID } from '../type/scalars';
  *
  * | JSON Value    | GraphQL Value        |
  * | ------------- | -------------------- |
+ * | null          | NullValue            |
```

Comment:
nitpick, could you put this entry at the bottom of the table, that way it's in the same order as the comments above valueFromAST.